desktop/cmp/form/FormField.js

```diff
@@ -0,0 +1,89 @@
+/*
+ * This file belongs to Hoist, an application development toolkit
+ * developed by Extremely Heavy Industries (www.xh.io | info@xh.io)
+ *
+ * Copyright © 2018 Extremely Heavy Industries Inc.
+ */
+import React, {Component} from 'react';
+import {elemFactory, HoistComponent} from '@xh/hoist/core';
+import {PropTypes as PT} from 'prop-types';
+import {HoistInput} from '@xh/hoist/cmp/form';
+import {formGroup, spinner} from '@xh/hoist/kit/blueprint';
+import {fragment, div, span} from '@xh/hoist/cmp/layout';
+import {throwIf} from '@xh/hoist/utils/js';
+import {isArray} from 'lodash';
+
+import './FormField.scss';
+
+/**
+ * Standardised wrapper around HoistInputs.
+ *
+ * Should receive a single HoistInput as a child element. If the HoistInput is bound
+ * to a model enhanced by `@FieldSupport`, FormField will automatically display a label,
+ * a required asterisk and validation state.
+ *
+ * Accepts any props supported by Blueprint's FormGroup.
+ */
+@HoistComponent
+export class FormField extends Component {
+
+    static propTypes = {
+        /** model to bind to */
+        model: PT.object,
+        /** name of property in model to bind to */
+        field: PT.string,
+        /** optional label for form field */
```

Comment:
We can lose "optional" here - I know it's scattered in a few places in the docs. They are all effectively optional. 

Sorry to nitpick - I need an emoji to preface annoying comments like this. Suggestions....maybe welcome